### ISSUE TITLE
iframe with focusable content does not have negative tabindex

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#2687]
We should get an implementation of this rule going fairly soon:
https://github.com/act-rules/act-rules.github.io/pull/1515/files

## Violations

```html
<iframe tabindex="-1" srcdoc="<a href='/'>Home</a>"></iframe>
<iframe tabindex="-1" srcdoc="<button>Save</button>"></iframe>
```

## Passes:

```html
<iframe tabindex="-1" srcdoc="<h1>Hello world</h1>"></iframe>
<iframe tabindex="-1" srcdoc="<a href='/' tabindex='-1'>Home</a>"></iframe>
<iframe tabindex="-1" width="1" height="1" srcdoc="<a href='/'>Home</a>"></iframe>
```

**Comment by padmavemulapati** [comment]
Verified with the latest axe-core- develop branch code base , I am unable to see the anything related to `frame-focusable` rule
![image](https://user-images.githubusercontent.com/47409579/107651353-6bdd4900-6ca5-11eb-8d11-b1b032b463eb.png)

**Comment by straker** [comment]
@padmavemulapati It looks like you triggered the `frame-tested` rule. The rule only works when the iframes also have axe-core in them. Could you try adding axe-core to each of the iframes and testing again?

**Comment by padmavemulapati** [comment]
Validated with the latest axe-core code base - seeing `frame-focusible-content` issue- when
``` index.html
<iframe title="frame" src="frame.html" tabindex="-1"></iframe>
<script src="/axe.js"></script>
```
```frame.html
<button>Save</button>
<script src="/axe.js"></script>
```

![image](https://user-images.githubusercontent.com/47409579/116051098-dcd1b180-a695-11eb-80d0-a929538c1532.png)
